apps/namada-interface/src/App/AccountOverview/DerivedAccounts/DerivedAccounts.tsx

```diff
@@ -135,6 +107,7 @@ const DerivedAccounts = (): JSX.Element => {
           .map((account) => {
             const { alias, address, isShielded } = account;
             const balance = balances[address];
+            const nativeBalance = typeof balance === "undefined" ? "-" : (balance?.[symbol]?.toString() || "0");
```

Comment:
The optional chaining on `balance` should be unneeded here.

Reply:
I think balance was undefined sometimes when I was testing it, but I will double check

Reply:
Ah right I'm checking that 😅